nucypher/characters/lawful.py

```diff
@@ -1284,6 +1288,10 @@ def handle_threshold_decryption_request(
         )
         return encrypted_response
 
+    def handle_threshold_signature_request(
```

Comment:
```suggestion
    def handle_1271_signature_request(
```
We may implement true threshold signatures in the future

Reply:
I suspect that the relevant logic will be dependent on the signing request blob received. We can always differentiate the two using, the contract metadata (info based on cohortId),  the `SigningRequest` object that is provided OR have a separate endpoint.

In any case, this was just starter code that I chatted with @KPrasch about, he's focused on this area and probably has his own changes. (ps @KPrasch you are welcome to add your changes to this branch, up to you)